packages/openssl/packaging

```diff
@@ -0,0 +1,11 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+mkdir ${BOSH_INSTALL_TARGET}/openssl
+echo "Extracting openssl"
+tar xzf openssl/openssl-1.0.2j.tar.gz
```

Comment:
@gberche-orange : Default behaviour of keepalived is to send it's logs to rsyslog. So we need to figure out how to trace the event of failover monitoring